src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -335,18 +337,10 @@ public void writeJson(Writer writer) {
     }
 
     private void logSize() {
-        int controlledBusCount = 0;
-        int controllerBusCount = 0;
-        for (LfBus bus : busesById.values()) {
-            if (bus.getControlledBus().isPresent()) {
-                controlledBusCount++;
-            }
-            if (!bus.getControllerBuses().isEmpty()) {
-                controllerBusCount++;
-            }
-        }
+        long remoteControllerBusCount = busesById.values().stream().filter(b -> b.isVoltageControllerEnabled() && !b.isVoltageControlled()).count();
```

Comment:
Not really equivalent to previous car as we iterate 2 time over all buses.
Maybe not an issue?